src/appmixer/github/list/ListIssues/transformers.js

```
@@ -0,0 +1,25 @@
+'use strict';
+
+/**
++ * Transforms a list of GitHub issues into an array suitable for selection UI components.
++ * Each issue is converted to an object with label (from title) and value (from number).
++ * 
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)